docs/user/openstack/README.md

```diff
@@ -287,6 +294,35 @@ Alternatively, if you don't control the DNS, you can add the hostnames in `/etc/
 <apps FIP> <app name>.apps.<cluster name>.<base domain>
 ```
 
+**WARNING:** *this workaround will make the API accessible only to the computer with these `/etc/hosts` entries. This is fine for your own testing (and it is enough for the installation to succeed), but it is not enough for a production deployment. In addition, if you create new OpenShift apps or routes, you will have to add their entries too.*
```

Comment:
To be extra explicit (and make it even clearer that the warning only applies to `/etc/hosts`), I'd add something like

```
[...] (as `/etc/hosts` does not support wildcard entries).
```

Reply:
I like it! It clarifies why you really want to use DNS rather than just a bunch of hostname entries.